extensions/amp-date-countdown/1.0/component.type.js

```diff
@@ -10,4 +10,4 @@
  *   render: (?RendererFunctionType|undefined),
  * }}
  */
-var DateCountdownPropsDef;
```

Comment:
While we're here, let's update to the following for consistency with other Bento typedefs:
```
/** @const */
var BentoDateCountdownDef = {};


/**
 * @typedef {{
 *   ...
 * }}
 */
BentoDateCountdownDef.Props;
```

Reply:
updated